app/buildOptions.gradle

```diff
@@ -0,0 +1,17 @@
+// This file written in Groovy is separated from the App-level build.gradle.kts.
+// This is only a temp solution for Kotlin DSL not being able to access the variables defined in
+// global gradle.properties file.
+android {
```

Comment:
Also, why not just add this to your `app/build.gradle.kts`?

Reply:
I couldn't access the variables stored in `/zluan/.gradle/gradle.properties` in `build.gradle.kts` for some reason. If I wrote it in groovy, it works fine. I probably missed something for kotlin dsl :/